src/test/java/by/issoft/pagefactory/AccountCreationPage.java

```diff
@@ -0,0 +1,128 @@
+
+public class AccountCreationPage {
+
+    WebDriver driver;
+
+    public AccountCreationPage(WebDriver driver) {
+        this.driver = driver;
+        PageFactory.initElements(driver, this);
+    }
+
+    @FindBy(id = "customer_firstname")
+    private WebElement firstNameField;
+
+    @FindBy(id = "customer_lastname")
+    private WebElement lastNameField;
+
+    @FindBy(id = "email")
+    private WebElement emailField;
+
+    @FindBy(id = "passwd")
+    private WebElement passwordField;
+
+    @FindBy(id = "address1")
+    private WebElement addressField;
+
+    @FindBy(id = "city")
+    private WebElement cityField;
+
+    @FindBy(id = "id_state")
+    private WebElement stateField;
+
+    @FindBy(id = "postcode")
+    private WebElement postalCodeField;
+
+    @FindBy(id = "id_country")
+    private WebElement coutryDropdown;
+
+    @FindBy(id = "phone_mobile")
+    private WebElement mobilePhoneField;
+
+    @FindBy(id = "alias")
+    private WebElement addressAliasField;
+
+    @FindBy(id = "submitAccount")
+    private WebElement registerButton;
+
+    @FindBy(xpath = "//*[@class='page-heading' and contains(text(), 'My account')]")
+    private WebElement myAccountPageTitle;
+
+    @FindBy(className = "account")
+    private WebElement userAccountName;
+
+
+    public AccountCreationPage enterFirstName(String firstName) {
+        firstNameField.sendKeys(firstName);
+        return this;
+    }
+
+    public AccountCreationPage enterLastName(String lastName) {
+        lastNameField.sendKeys(lastName);
+        return this;
+    }
+
+    public AccountCreationPage enterEmail(String email) {
+        emailField.sendKeys(email);
+        return this;
+    }
+
+    public AccountCreationPage enterPassword(String password) {
+        passwordField.sendKeys(password);
+        return this;
+    }
+
+    public AccountCreationPage enterAddress(String address) {
+        addressField.sendKeys(address);
+        return this;
+    }
+
+    public AccountCreationPage enterCity(String city) {
+        cityField.sendKeys(city);
+        return this;
+    }
+
+    public AccountCreationPage selectState(String state) {
+        Select stateDropdown = new Select(stateField);
+        stateDropdown.selectByVisibleText(state);
+        return this;
+    }
+
+    public AccountCreationPage enterPostalCode(String city) {
+        postalCodeField.sendKeys(city);
+        return this;
+    }
+
+    public AccountCreationPage enterMobilePhone(String mobilePhone) {
+        mobilePhoneField.sendKeys(mobilePhone);
+        return this;
+    }
+
+    public AccountCreationPage enterAddressAlias(String addressAlias) {
+        addressAliasField.sendKeys(addressAlias);
+        return this;
+    }
+
+    public AccountCreationPage clickRegisterButton() {
+        registerButton.click();
+        return this;
+    }
+
+    public AccountCreationPage checkMyAccountPageOpened() {
+        Assertions.assertTrue(myAccountPageTitle.isDisplayed());
```

Comment:
Assertions should be move from page classes. It should be only in test classes. In the method you can do check on equals and return  Boolean result from the method. In the test you will check on expected boolean value